src/mrpro/data/IHeader.py

```diff
@@ -38,6 +48,48 @@ class IHeader(MoveDataMixin):
     tr: torch.Tensor | None
     """Repetition time [s]."""
 
+    phase_dir: SpatialDimension[torch.Tensor] | None
+    """Directional cosine of phase encoding (2D)."""
+
+    position: SpatialDimension[torch.Tensor] | None
+    """Center of the excited volume, in LPS coordinates relative to isocenter [m]."""
+
+    read_dir: SpatialDimension[torch.Tensor] | None
+    """Directional cosine of readout/frequency encoding."""
+
+    slice_dir: SpatialDimension[torch.Tensor] | None
+    """Directional cosine of slice normal, i.e. cross-product of read_dir and phase_dir."""
+
+    patient_table_position: SpatialDimension[torch.Tensor] | None = None
+    """Offset position of the patient table, in LPS coordinates [m]."""
+
+    n_coils: int | None = None
```

Comment:
Why? Shouldn't this be detected by the data?
This might give rise to inconsistencies or needs updating on subsetting.
I would say this is not worth the effort.

Reply:
remove also from KHeader